apps/cli/src/utils/prompt.ts

```diff
@@ -17,6 +17,17 @@ export async function querySelect<T>(
 
 export const queryInput = async (message: string) => input({ message });
 
+export const queryNumber = async (message: string) => {
+  const str = await input({ message });
+  const num = parseInt(str, 10);
+
+  if (Number.isNaN(num)) {
+    throw new Error('Invalid number');
+  }
```

Comment:
It's seems too unforgiving to throw an error on invalid input, perhaps we should let the user retry instead?